### DISCUSSION TITLE
유니티에 제페토 UI가 두개가 생겼어요

### DISCUSSION THREAD

**Discussion by SaesolPark** [naverz/zepeto-studio-kor#843] [5. Q&A]
![KakaoTalk_20220606_131413496](https://user-images.githubusercontent.com/80307607/172094617-848b89e2-7560-48d9-91b7-919702a89f48.png)
저번에 유니티에서 제페토 UI가 사라져서 리임포트올을 했는데요 그때도 안생기다가 후에 두개가 생겨버렸어요
그래서 제페토 스크립트가 중복되서 실행이 안되네요 어떻게 해야하나요? 지우는 방법도 모르겠어요

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 해당 건 확인해 보겠습니다.
추후 오류 상황에 대한 제보는 이슈 게시판에 올려주시면 이슈 관리 및 진행 상황 업데이트에 도움이 됩니다.
감사합니다.
https://github.com/naverz/zepeto-studio-kor/issues

**Reply by Zepeto-TechEvangelist** [reply]
개발 환경 및 제페토 월드패키지 버전은 어떻게 되나요? 
https://github.com/naverz/zepeto-studio-kor/discussions/828 처음에 컴퓨터를 켰을때 뜬 오류창은 어떤 내용이었는지도 알려주시면 감사 드리겠습니다.
